job/dhcp.go

```diff
@@ -70,23 +70,43 @@ func (j Job) configureDHCP(ctx context.Context, rep, req *dhcp4.Packet) bool {
 	}
 
 	if dhcp.SetupPXE(ctx, rep, req) {
+		arch := "x86"
+		firmware := "bios"
+
 		isARM := dhcp.IsARM(req)
 		if dhcp.Arch(req) != j.Arch() {
 			span.AddEvent(fmt.Sprintf("arch mismatch: got %q and expected %q", dhcp.Arch(req), j.Arch()))
 			j.With("dhcp", dhcp.Arch(req), "job", j.Arch()).Info("arch mismatch, using dhcp")
 		}
+		if isARM {
+			arch = "arm"
```

Comment:
might be useful to make these values (`arm`, `hua`, `2a2`, etc) constants?

Reply:
I'm not too convinced that going from bare strings to a token is very helpful in this case. `arch` isn't going to be const, and I think that 

```go
arch = "arm"
...
case arch == "arm"
```
just reads better than
```go
arch = archARM
...
case arch == archARM
```.

The arch is strings are used in just 2 locations (setting/testing) and I don't think would be worth it to have an indentifier to avoid a little repitition.